### PR TITLE
Bring 3.4 whatsnew up-to-date

### PR DESCRIPTION
**Pull request by nickva**
From June [1] until now

[1] https://github.com/apache/couchdb/commit/62b501873b7e7990a64783c134dd8f1fca77780f

### CODE REVIEW

File: src/docs/src/whatsnew/3.4.rst [hunk 1]
```diff
@@ -215,6 +228,47 @@ Performance
 Bugfixes
 --------
 
+* :ghissue:`5223`, :ghissue:`5228`, :ghissue:`5226` Fix handling IPv6 addresses
+  for ``_session`` endpoints in replicator.
+* :ghissue:`5191`, :ghissue:`5193`: Fix error loop with system freeze when
+  removing a node from a cluster.
+* :ghissue:`5188`: Fix units for replicator ``cluster_start_period`` config
+  setting.
+* :ghissue:`5185`: Use an explicit message for replicator doc processor.
```
Comment (big-r81) [inline src/docs/src/whatsnew/3.4.rst:237]: Remove the '.' after processor?

File: src/docs/src/whatsnew/3.4.rst [hunk 2]
```diff
@@ -215,6 +228,47 @@ Performance
...
+* :ghissue:`5205`: Cleanly abort responses when path doesn't start with slash.
+* :ghissue:`5204`, :ghissue:`5203`, :ghissue:`5200`, :ghissue:`5201`: Fix
+  continuous changes feeds with a limit greater than total.
+* :ghissue:`5169`: Make sure we never get an inconsistent ``couch_lru`` cache.
+* :ghissue:`5167`: Remove unused ``close_lru`` ``gen_server`` call.
+* :ghissue:`5160`: Ensure we run fabric worker cleanup in more cases.
+* :ghissue:`5158`: Fix PowerShell ``PSScriptAnalyzer`` warnings.
+* :ghissue:`5153`, :ghissue:`5156`: Upgrade recon and fix Erlang/OTP 27
+  compiler warnings.
+* :ghissue:`5154` Replace ``0/1`` to ``false/true`` for config keys.
```
Comment (big-r81) [inline src/docs/src/whatsnew/3.4.rst:251]: missing ':' after gh issue number.

File: src/docs/src/whatsnew/3.4.rst (hunk 1 above)
Comment (nickva) [reply src/docs/src/whatsnew/3.4.rst:237]: Definitely. Thanks!

File: src/docs/src/whatsnew/3.4.rst (hunk 2 above)
Comment (nickva) [reply src/docs/src/whatsnew/3.4.rst:251]: Thanks!

Comment (nickva) [reply src/docs/src/whatsnew/3.4.rst]: Updated

File: src/docs/src/whatsnew/3.4.rst (hunk 1 above)
Comment (nickva) [reply src/docs/src/whatsnew/3.4.rst]: Updated.

File: src/docs/src/whatsnew/3.4.rst [hunk 3]
```diff
@@ -215,6 +228,47 @@ Performance
 Bugfixes
 --------
 
+* :ghissue:`5223`, :ghissue:`5228`, :ghissue:`5226` Fix handling IPv6 addresses
```
Comment (big-r81) [inline src/docs/src/whatsnew/3.4.rst:231]: Missing ':' after issues.

Comment (nickva) [reply src/docs/src/whatsnew/3.4.rst]: Thank you. Updated.

**Review by big-r81 — APPROVED** [review]
Nice work Nick! Thank you.

**Pull request merged**